pkg/kubelet/server/auth.go

```diff
@@ -63,7 +65,7 @@ func isSubpath(subpath, path string) bool {
 //	/stats/*   => verb=<api verb from request>, resource=nodes, name=<node name>, subresource=stats
 //	/metrics/* => verb=<api verb from request>, resource=nodes, name=<node name>, subresource=metrics
 //	/logs/*    => verb=<api verb from request>, resource=nodes, name=<node name>, subresource=log
-func (n nodeAuthorizerAttributesGetter) GetRequestAttributes(u user.Info, r *http.Request) authorizer.Attributes {
+func (n nodeAuthorizerAttributesGetter) GetRequestAttributes(u user.Info, r *http.Request) []authorizer.Attributes {
```

Comment:
update the godoc to at least mention the other subpaths and the feature behavior that changes the returned value

Reply:
Done.